telemetry/admin-metrics-export/src/main/scala/io/buoyant/telemetry/admin/AdminMetricsExportTelemeter.scala

```diff
@@ -0,0 +1,123 @@
+package io.buoyant.telemetry.admin
+
+import com.fasterxml.jackson.core.JsonFactory
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter
+import com.twitter.finagle.http.{MediaType, Request, Response}
+import com.twitter.finagle.stats.NullStatsReceiver
+import com.twitter.finagle.tracing.NullTracer
+import com.twitter.util._
+import io.buoyant.admin.Admin
+import io.buoyant.telemetry.Metric.{Counter, Gauge, HistogramSummary, Stat}
+import io.buoyant.telemetry.{Metric, MetricsTree, Telemeter}
+import java.io.OutputStream
+import java.util.concurrent.atomic.AtomicBoolean
+import scala.collection.mutable
+
+/**
+ * AdminMetricsExportTelemeter provides the /admin/metrics.json admin endpoint, backed by a
+ * MetricsTree.  It does not provide a StatsReciever or Tracer.  Histograms are snapshotted into
+ * summaries at a regular interval and the values served on /admin/metrics.json are taken from the
+ * most recent summary snapshot.  Counter and gauge values are served live.
+ */
+class AdminMetricsExportTelemeter(
+  metrics: MetricsTree,
+  snapshotInterval: Duration,
+  timer: Timer
+) extends Admin.Handler with Telemeter with Admin.WithHandlers {
```

Comment:
What's the thought on making this extend Admin.Handler directly?  It might be nicer to have a scoped member `val handler: Admin.Handler = new Service[...] { ... }`